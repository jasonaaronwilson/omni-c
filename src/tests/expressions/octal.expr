0o1234567;
