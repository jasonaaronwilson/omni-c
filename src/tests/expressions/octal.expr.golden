{
    tag = PARSE_NODE_LITERAL
    tokens = [
        "0o1234567"
    ]
}

// C Output
0o1234567
